src/test/java/net/minecraftforge/debug/client/CustomizeGuiOverlayEventTest.java

```diff
@@ -0,0 +1,35 @@
+package net.minecraftforge.debug.client;
+
+import net.minecraftforge.api.distmarker.Dist;
+import net.minecraftforge.client.event.CustomizeGuiOverlayEvent;
+import net.minecraftforge.eventbus.api.EventPriority;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fml.common.Mod;
+
+@Mod(CustomizeGuiOverlayEventTest.MODID)
+public class CustomizeGuiOverlayEventTest
+{
+    private static final boolean ENABLED = false;
+
+    public static final String MODID = "custom_gui_event_test";
+    @Mod.EventBusSubscriber(value = Dist.CLIENT, modid = MODID, bus = Mod.EventBusSubscriber.Bus.FORGE)
```

Comment:
Its better to manually register things using the ENABLED flag in test mods instead of using eventbussubscriber. As that will always subscribe things.

Reply:
fixed